invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/invitationsModal/GroupTabPane.js

```diff
@@ -99,9 +100,12 @@ export class GroupTabPane extends Component {
             floated="left"
             onClick={modalClose}
           />
-          <Trans key="communityInviteMembersSelected" count={selectedCount}>
-            You are about to invite {{ selectedCount }} groups
-          </Trans>
+          {selectedCount > 0 && (
+            <Trans key="communityInviteMembersSelected" count={selectedCount}>
+              You are about to invite {{ selectedCount }} groups
```

Comment:
Could you please fix another thing?
Groups are added, not invited.

* change the text to `You are about to add {{ selectedCount }} groups`
* change the button below `Invite` -> `Add`
* anything else?
See mock for reference